phalcon/db/adapterinterface.zep

```diff
@@ -95,7 +95,7 @@ interface AdapterInterface
 	 * @param 	int number
 	 * @return 	string
 	 */
-	public function limit(var sqlQuery, number);
+	public function limit(string sqlQuery, int number);
```

Comment:
Theoretically, specific implementations can use `sqlQuery` as a Query object

Reply:
i did this because of https://github.com/phalcon/cphalcon/blob/master/phalcon/db/adapter.zep#L677 . 
should I update Db\Adapter class to accept `var sqlQuery` cuz i am not sure what is the correct approach